APE0.rst

```diff
@@ -0,0 +1,393 @@
+  infrastructure telecons, proposal-related telecons, and Astropy Learn
+  telecons.
+* Has commit rights to at least one repository in the astropy GitHub
+  organization which is either the core package, a coordinated package,
+  an infrastructure package, or an Astropy Learn-related repository,
+  and has actively used those commit rights in the last two years prior to the
+  acceptance of this APE.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can be nominated as a Voting Member by providing evidence of meeting the
+requirements laid out in the Membership section above. Both self-nominations and
+nominations by others are allowed. The Voting Members are expected to make their
+decisions based on the candidate’s adherence to the membership criteria, above.
+The name of the nominee will be known to Voting Members but will not be shared
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will put it to a vote among active Voting Members. Votes for adding
+  voting members will be carried out at most twice a year to avoid too many
+  elections. The voting dates will be announced with at least four weeks notice,
+  and additional nominations will be accepted until two weeks before the
+  election date. The voting will be open for at least two weeks. To successfully
+  gain an appointment as a Voting Member, the nominee must receive at least four
+  positive votes, as long as that constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+At least one round of voting for adding voting members should be carried out
+before the initial election of the Coordination Committee.
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as *emeritus*. Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the *emeritus* category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to *emeritus*. To record and honor their contributions,
+*emeritus* Voting Members will continue to be listed. *Emeritus* Voting Members
+are not able to participate in votes.
+
+Removing Voting Members
```

Comment:
These are not clear from the wordings:

* So, when someone is removed this way, they don't go to "emeritus" list, or do they?
* Also, I am guessing they are not allowed to re-run to be voting member again in the future? Or is that not true?
* Since non-voting member can run for CoCo, are they still qualified to run for CoCo?

Reply:
Hmm.  Should we say the voting members hold a second vote to bar from future office if convincted? (too soon?)

More seriously, all good points. Definitely not for "emeritus" (that came up in the WG in passing), but I'm not sure the other two were discussed.  

My feeling is that there's no need to make a rule against running for CoCo, since it's the *same people* who voted them out who'd then vote in the CoCo.

Less clear to me is about voting membership, again, it's the same group, so I guess there's no harm in allowing it, but maybe we should say "at least a year" or similar to prevent "election spamming"?

I'll update for the first and third of these, but might be good to get your thoughts, @pllim or @astrofrog on this second item (one year)

Reply:
The CoCo discussed this and agreed we think it's best to leave the latter two un-specified since hopefully this is an edge case, and the voting for CoCo or to be re-added as a member is unlikely to succeed anyway.  So better to "never say never" (@astrofrog's words).

Reply:
If I have learned anything in the past 5 years, is that any unwritten norm relying on good faith will be broken at some point... but since this has been discussed and decided upon otherwise, I rest my case.